src/sage/algebras/fusion_rings/fusion_double.py

```diff
@@ -0,0 +1,891 @@
+
+        EXAMPLES::
+
+            sage: FusionDouble(DiCyclicGroup(4)).group()
+            Diyclic group of order 16 as a permutation group
+        """
+        return self._G
+
+    def get_fmatrix(self, *args, **kwargs):
+        r"""
+        Construct an :class:`FMatrix` factory to solve the pentagon and
+        hexagon relations and organize the resulting F-symbols.
+
+        EXAMPLES::
+
+            sage: f = FusionDouble(SymmetricGroup(3)).get_fmatrix(); f
+            F-Matrix factory for The Fusion Ring of the Drinfeld Double of Symmetric group of order 3! as a permutation group
+        """
+        if not hasattr(self, 'fmats') or kwargs.get('new', False):
+            kwargs.pop('new', None)
+            from sage.algebras.fusion_rings.f_matrix import FMatrix
+            self.fmats = FMatrix(self, *args, **kwargs)
+        return self.fmats
+
+    class Element(CombinatorialFreeModule.Element):
+        def is_simple_object(self):
+            r"""
+            Determine whether ``self`` is a simple object (basis element) of the fusion ring.
+
+            EXAMPLES::
+
+                sage: H = FusionDouble(CyclicPermutationGroup(2), prefix="g", inject_variables=True)
+                sage: [x.is_simple_object() for x in [g0, g1, g0+g1]]
+                [True, True, False]
+            """
+            return len(self._monomial_coefficients) == 1
+
+        def g(self):
+            r"""
+            The data determining a simple object consists of a conjugacy
+            class representative `g` and an irreducible character `\chi` of
+            the centralizer of `g`.
+
+            Returns the conjugacy class representative of the underlying
+            group corresponding to a simple object. See also :meth:`char`.
+
+            EXAMPLES::
+
+                sage: G = QuaternionGroup()
+                sage: H = FusionDouble(G, prefix="e", inject_variables=True)
+                sage: e10.g()
+                (1,3)(2,4)(5,7)(6,8)
+                sage: e10.char()
+                Character of Subgroup generated by [(1,2,3,4)(5,6,7,8), (1,5,3,7)(2,8,4,6)] of
+                 (Quaternion group of order 8 as a permutation group)
+            """
+            return self.parent()._elt[self.support_of_term()]
+
+        def char(self):
+            r"""
+            Return the character `\chi` corresponding to ``self``.
+
+            The data determining a simple object consists of a conjugacy
+            class representative `g` and an irreducible character `\chi` of
+            the centralizer of `g`.
+
+            .. SEEALSO:: :meth:`g`
+
+            EXAMPLES::
+
+                sage: G = DihedralGroup(5)
+                sage: H = FusionDouble(G, prefix="f", inject_variables=True)
+                sage: f10.g()
+                (1,2,3,4,5)
+                sage: f10.char()
+                Character of Subgroup generated by [(1,2,3,4,5)] of (Dihedral group of order 10 as a permutation group)
```

Comment:
I'd suggest to break long output lines like this one so that the example can be read in the HTML documentation without need for horizontal scrolling

Reply:
> I'd suggest to break long output lines like this one so that the example can be read in the HTML documentation without need for horizontal scrolling 

OK, I did this in commit c9c532d.